cache/class-vary-cache.php

```diff
@@ -457,7 +463,7 @@ private static function parse_group_cookie() {
 	 */
 	private static function stringify_groups() {
 		if ( empty( self::$groups ) ) {
-			return;
+			return null;
```

Comment:
Here and below: `Refactor this function to use "return" consistently`. Returning `null` is in line with what the doc block says: `@return null|string`

Reply:
Might be better to see where this private function is called, and to return an empty string so at least the return type is consistent.

Reply:
Will do that in another PR, thanks